pekko-sample-distributed-workers-scala/src/main/scala/worker/Main.scala

```diff
@@ -80,13 +80,13 @@ object Main {
 
   def config(port: Int, role: String): Config =
     ConfigFactory.parseString(s"""
-      akka.remote.artery.canonical.port=$port
-      akka.cluster.roles=[$role]
+      pekko.remote.artery.canonical.port=$port
+      pekko.cluster.roles=[$role]
     """).withFallback(ConfigFactory.load())
 
   /**
     * To make the sample easier to run we kickstart a Cassandra instance to
-    * act as the journal. Cassandra is a great choice of backend for Akka Persistence but
+    * act as the journal. Cassandra is a great choice of backend for Pekko Persistence but
```

Comment:
could you prefer `Apache Pekko` to `Pekko`? In a paragraph, where you continuously mention Pekko, it is ok to start with `Apache Pekko` and switch to `Pekko` but in short paragraphs, stick to `Apache Pekko`.

Reply:
Lightbend didn't care but as a member of the ASF, I do - could you cal it Apache Cassandra?

Reply:
updated all instances of Pekko and Cassandra acc. to guidance above